pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -772,40 +839,59 @@ func (p *MasqueradePodInterface) createNatRulesUsingIptables() error {
 	return nil
 }
 
-func (p *MasqueradePodInterface) createNatRulesUsingNftables() error {
-	err := Handler.NftablesLoad("ipv4-nat")
-	if err != nil {
-		return err
+func getGatewayByProtocol(p *MasqueradePodInterface, proto iptables.Protocol) string {
```

Comment:
can you store the `proto` in the struct and just add a `getGateway()` method to `MasqueradePodInterface`?